<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shutDownDialog</class>
 <widget class="QDialog" name="shutDownDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>关机</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 30, 41);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="shutdownBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>331</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rebootBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>331</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>331</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(66, 79, 131);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>376</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>先关闭系统后，用钥匙彻底关闭电源！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>returnBtn</sender>
   <signal>clicked()</signal>
   <receiver>shutDownDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
